docs/docs/guides/reverse_proxy.md

```diff
@@ -0,0 +1,66 @@
+---
+id: reverse_proxy
+title: Setting up a Reverse Proxy
+---
+This guide outlines the configuration steps needed to expose your Frigate UI to the internet in a secure manner.
+A common way of accomplishing this is to use a reverse proxy webserver between your router and your Frigate instance.
+
+A reverse proxy accepts HTTP requests the public internet and redirects them transparently to an internal webserver on your network.
+The suggested steps are:
+- **Configure** a 'proxy' HTTP webserver (such as [Apache2](https://httpd.apache.org/docs/current/)) and only expose ports 80/443 from this webserver to the internet
```

Comment:
I think we should mention more than just one reverse proxy here if this is going to be a generic reverse proxy guide

Reply:
I have done this, but I don't have the knowledge to add others. I've invited others to contribute in the text.